.circleci/config.yml

```diff
@@ -67,11 +67,17 @@ commands:
             - /home/circleci/.composer/cache/
       - store_artifacts:
           path: ./artifacts
-      - slack/status:
-          fail_only: true
-          failure_message: ':red_circle: M2: A $CIRCLE_JOB job has failed!'
-          only_for_branches: master
-          webhook: $SLACK_GREENKEEPER_WEBHOOK
+      - when:
+          condition:
+            or:
+              - equal: ["jz/rebase-custom1", << pipeline.git.branch >>]
+              - equal: ["jz/rebase-custom2", << pipeline.git.branch >>]
+              - equal: ["jz/rebase-custom3", << pipeline.git.branch >>]
```

Comment:
These branch names should come from the config file instead of being hardcoded here